docs/en/ingest-management/overview.asciidoc

```diff
@@ -40,6 +40,27 @@ It can be started from any available x64 architecture {agent} artifact.
 
 For more information, see <<fleet-server>>.
 
+[discrete]
+[[package-registry-intro]]
+== {package-registry}
+
+The {package-registry} is an online package hosting service for the integrations
+available for selection using the {fleet} UI in {kib}.
```

Comment:
Getting nit-picky here, but `"for the integrations available for selection using the"` didn't sound quite right.

```suggestion
The {package-registry} is an online package hosting service for the integrations
available in the {fleet} UI in {kib}.
```

Reply:
Thanks!